### PR TITLE
Add mlocati/ip-lib

### PR DESCRIPTION
**Pull request by come-nc**
Add dependency to mlocati/ip-lib to be able to parse IPs in non-decimal formats.

Signed-off-by: Côme Chilliet <come.chilliet@nextcloud.com>

### CODE REVIEW

File: mlocati/ip-lib/composer.json [hunk 1]
```diff
@@ -0,0 +1,60 @@
+{
```
Comment (nickvergessen) [inline mlocati/ip-lib/composer.json:1]: gitignore please

Same for the readme

**Review by ChristophWurst — APPROVED** [review]

**Review by nickvergessen — APPROVED** [review]

File: mlocati/ip-lib/ip-lib.php [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?php
+
+spl_autoload_register(
```
Comment (nickvergessen) [inline mlocati/ip-lib/ip-lib.php:3]: actually this should not be here? Autoloading is done via composer?

**Pull request merged**